src/application/VariationWriter.ts

```diff
@@ -107,11 +116,18 @@ export class VariationWriter {
     }
 
     if (oaOperation && tests?.contractTests) {
+      // Set target OpenAPI response
+      let targetOaResponse = variation?.openApiResponse
+      if (!variation?.openApiResponse && variationMeta?.openApiResponse) {
```

Comment:
can use `targetOaResponse` here

Reply:
just seems weird we check for it and then overwrite it, but maybe I'm missing context

Reply:
ah nvm - I get it

Reply:
I need to check, if "variation.openApiResponse" is NOT set, then I'm allowed to take the "variationMeta?.openApiResponse" as "targetOaResponse"

Reply:
Corrected